apps/nextjs-website/react-components/types/FramedVideo/FramedVideo.types.ts

```diff
@@ -0,0 +1,39 @@
+import { SectionProps } from '../common/Common.types';
+
+export interface FramedVideoProps extends SectionProps {
+  theme: 'light' | 'dark';
+  videoURL?: string | null;
```

Comment:
```suggestion
  videoURL?: string;
```
Doesn't make sense to support null.

Either the string exists or it's not passed at all.